tests/embeddings.rs

```diff
@@ -486,6 +486,63 @@ fn test_image_embedding_model() {
         });
 }
 
+#[test]
+#[ignore]
```

Comment:
Why ignore?

Reply:
Because its already slightly tested in test_image_embedding_model by means of list_supported_models(), and didnt want to pollute your CI tests. But can be enabled if you want.

Reply:
Thanks. Makes sense.